crates/figma_import/src/toolkit_font_style.rs

```diff
@@ -100,8 +100,10 @@ impl PartialEq for FontWeight {
                 NumOrVar::Num(o) => f32_eq(n, o),
                 _ => false,
             },
-            NumOrVar::Var(v) => match &other.0 {
-                NumOrVar::Var(o) => v == o,
+            NumOrVar::Var { id: id1, fallback: num } => match &other.0 {
```

Comment:
This is the existing structure of the function, but there might be a cleaner way to do it like 
```
match (&self.0, &other.0) {
  (NumOrVar::Num(&a, _), NumOrVar::Num(&b, _)) => a == b,
...
```

Reply:
Done.